src/content/en/2020/caching.md

```diff
@@ -21,3 +21,1376 @@ queries: 20_Caching
 #featured_stat_label_3: TODO
 unedited: true
 ---
+
+<!-- Copy and paste the converted output. -->
```

Comment:
There's a lot of boilerplate in here (comments, doc metadata, etc) and what seems like a lot of HTML rather than markdown. See the [2019 caching.md](https://raw.githubusercontent.com/HTTPArchive/almanac.httparchive.org/main/src/content/en/2019/caching.md) for comparison.

@roryhewitt are you comfortable going through this and converting into pure markdown as much as possible? It's ok for some things like tables to be HTML if needed, but markdown is preferred for almost everything else.